src/services/handlers/searchResultsApi.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiResponse } from '../../Models/ApiResponse';
+import { geoNetworkClient } from '../../config/geoNetworkClient';
+import { BoolModel, Must, Query, Querystring, SearchRequest } from '../../Models/SearchRequest';
+
+const url = 'http://20.77.9.229:8080/geonetwork/srv/api/search/records/_search?bucket=s101';
```

Comment:
please move this url to a config constant and use it. in future it would be env specific